readme.md

```diff
@@ -163,6 +165,14 @@ You can set `data-image` and `data-description` attributes to set up an image an
 <div class="vkontakte" data-image="https://placekitten.com/200/400" data-description="Check this out">Share</div>
 ```
 
+### Viber
+
+You can set `data-comment` attribute to define a text of the message.
```

Comment:
Better: "to specify some text for the shared message".
Also, is it gonna be added to the message or substituted? Please document this too.